lib/src/responsive_scaled_box.dart

```diff
@@ -40,7 +44,7 @@ class ResponsiveScaledBox extends StatelessWidget {
               padding: scaledViewPadding, insets: scaledViewInsets);
 
           Widget childHolder = FittedBox(
-            fit: BoxFit.fitWidth,
+            fit: BoxFit.cover,
```

Comment:
We can't change this for everyone.

Reply:
I fixed and return that back to `Boxfit.fitWidth`, actually in my tests it doesn't make any difference to output result because of respecting to aspect ratio.